pandas/tests/util/test_testing.py

```diff
@@ -499,10 +517,12 @@ def _assert_not_equal(self, a, b, **kwargs):
     def test_equal(self):
         self._assert_equal(Series(range(3)), Series(range(3)))
         self._assert_equal(Series(list('abc')), Series(list('abc')))
+        self._assert_equal(Series(list(u'áàä')), Series(list(u'áàä')))
```

Comment:
can you add a test where left is unicode and right is non-unicode (but string)

Reply:
👍 1f7e231